include/signature_subpacket/embedded.h

```diff
@@ -42,13 +42,13 @@ namespace pgp::signature_subpacket {
              *
              *  @param  value      The value to store
              */
-            embedded(contained_t value);
+            explicit embedded(contained_t value);
 
             /**
              *  Copy and move constructors
              */
             embedded(const embedded &other);
-            embedded(embedded &&other) = default;
+            embedded(embedded &&other) noexcept = default;
```

Comment:
Not necessary, it's already `noexcept` because `std::unique_ptr::unique_ptr(std::unique_ptr&&)` is `noexcept`.

Reply:
Reverted